pkgs/development/python-modules/azure-mgmt-automation/default.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  msrest,
+  azure-common,
+  azure-mgmt-core,
+}:
+
+buildPythonPackage rec {
+  pname = "azure-mgmt-automation";
+  version = "1.0.0";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```

and add `build-system = [ setuptools ];`